docs/FAQ.md

```diff
@@ -120,4 +125,4 @@ You can stop lending when you choose. Withdrawing early may result in a loss of
 The portfolio value is calculated as `(fyDai * APR) / (days to maturity / 365)`
 
 ### I lent some Dai, and the Current Value is show as less that I invested, why?
-when you lend you are selling Dai to the pool, and getting fyDai back. When you close your position you are selling the fyDai that you got before, and getting Dai. Both operations in the pool involve paying a fee of around 5% on the interest rate. If that specific maturity has an interest rate of a 3%, the fee would be of about (0.03 * 0.05 * trade size) in Dai. The portfolio value is thus reduced by the trading fee, and the Current Value reflects the return on selling the fyDai in the pool, which reflects the fee, interest rate changes, and slippage.
\ No newline at end of file
+when you lend you are selling Dai to the pool, and getting fyDai back. When you close your position you are selling the fyDai that you got before, and getting Dai. Both operations in the pool involve paying a fee of around 5% on the interest rate. If that specific maturity has an interest rate of a 3%, the fee would be of about (0.03 * 0.05 * trade size) in Dai. The portfolio value is thus reduced by the trading fee, and the Current Value reflects the return on selling the fyDai in the pool, which reflects the fee, interest rate changes, and slippage.
```

Comment:
I don't think this is right: "(0.03 * 0.05 * trade size)".  The fee is effectively an interest rate so shouldn't it be calculated like one? Where is the time component?